### ISSUE TITLE
implement Illuminate\Support\MessageBag for integration of Laravel's validation

### ISSUE THREAD

**Issue by 18601673727** [codecourse/notify#7]

**Comment by ctf0** [comment]
can u explain more of what r u exactly after ?, maybe like a case scenario

**Comment by 18601673727** [comment]
@ctf0  This's a long time ago issue, Sorry I already forgot the details but `notify` is using plain `Array` or `Object` pass to view layer. Without Laravel's `MessageBag` class implement means you can't integrate this with Laravel's validation or collection mutation staff etc.

**Comment by ctf0** [comment]
yes i understand, but can u show me a code example like when a validation error happens, whats the code u want to write in order to catch it in the view, cuz i already have an idea but not sure yet of what exactly u have in mind.
